src/V3EmitCBase.h

```diff
@@ -40,7 +40,7 @@ class EmitCParentModule final {
     EmitCParentModule();
     VL_UNCOPYABLE(EmitCParentModule);
 
-    static const AstNodeModule* get(const AstNode* nodep) {
+    static const AstNodeModule* get(const AstNode* nodep) VL_MT_SAFE {
```

Comment:
```suggestion
    static const AstNodeModule* get(const AstNode* nodep) VL_MT_SAFE_STABLE_TREE {
```

op4p() should be VL_MT_STABLE_TREE, so this should be warned by the attribute checks.